orchestrator/graphql/schema.py

```diff
@@ -150,12 +147,33 @@ async def get_context(custom_context=Depends(custom_context_dependency)) -> Orch
     return custom_context
 
 
-def create_graphql_router(query: Any = Query, mutation: Any = Mutation) -> OrchestratorGraphqlRouter:
+GRAPHQL_MODELS_INITIAL = dict(GRAPHQL_MODELS)
```

Comment:
`GRAPHQL_MODELS_INITIAL` will always have the same content as `GRAPHQL_MODELS`, so I don't think it has any purpose?

Reply:
it doesn't, because the `GRAPHQL_MODELS_INITIAL` only has all the models before starting the auto generate. the `GRAPHQL_MODELS` includes the auto registered models and the auto generate wouldn't re-create the models that already exist in the `GRAPHQL_MODELS`.

Reply:
In the create_graphql_router it is referenced to because of `models = GRAPHQL_MODELS_INITIAL`, after which `models` is modified. Finally there is `GRAPHQL_MODELS.update(models)` at which point they contain the same data